### ISSUE TITLE
TumblThree can't crawl any blog

### ISSUE THREAD

**Issue by amorakfd** [TumblThreeApp/TumblThree#145]
Hello,

I use TumblThree-v1.0.11.13 version and this is issue I encounter :

This happens with all tumblr blogs, hidden or not, NFWS or not. 
Adding a blog in list works well but nothing appears in the "Number of downloads" column.

- When I try to crawl via API, the blog well appears in queue on the bottom right corner but the crawl doesn't start. When I ask any "blog online status" the application crashes.

- Trying to crawl via SVC, the information "Hidden blog" (although it's not) is added. When I try to add to queue and start crawl, I only see "Evaluated 1 tumblr blog sites" instead of datas downloading! and the crawl ends with no data downloaded.

Do I have to change anything in the application settings? I don't know what may cause that. Please, does anyone can help?

**Comment by thomas694** [comment]
Hello,
please use the newest version. Your version is a bit outdated and we fixed several issues in the last versions.
As the current version runs fine just now, I'm not expecting that you only experience a temporary problem. But you can wait, if you prefer.

**Comment by amorakfd** [comment]
Hello,

Thanks Thomas for your reply,

I've downloaded the latest version and already installed it but the program doesn't launch, neither on windows 7 or windows 10. Nothing happens when I click on the exe file. Please do you have a solution to cop with it? Do I need to install any additional *.dll or else?

Thanks a lot for your help,

**Comment by amorakfd** [comment]
Hello Thomas,

Thanks for your reply,

It's probably a question of that out of date version I used, I hope so.
I've already tried to use the latest version available to make sure.

The latest version : TumblThree-v1.4.1 is already installed on my computers
working under two different operating systems. But the program doesn't
launch under  windows 7 or windows 10. I have tried all I know to fix it
without success. Please, can you help me how to open it successfully?

All advice is welcome?

Frédéric

**Comment by thomas694** [comment]
For starting and running it doesn't need any additional dlls except the "Visual C++ Redistributable" package mentioned on the download/release page, if not yet available in the system. That one is necessary for the authenticate and privacy consent functionality (buttons in the settings dialog) to work.

Do you have any security/firewall software installed which could prevent executing the program? Can you please check the windows event logs for a hint what happens when you try to start the exe?
As the program doesn't even start I think our log file (folder %LOCALAPPDATA%\TumblThree) won't have any hints.

**Comment by amorakfd** [comment]
I check for anything which could help us to find out why the new program doesn't launch on my computer, in the tumblthree local data or windows event log with no result, sorry.

All I can say is that all versions before TumblThree-v1.3.0 release were working perfectly well. The issue happened with v1.3.0 and the upper releases.

The program simply doesn't launch. When I double click on the *.exe file, the Windows hourglass can be seen during few seconds and vanishes but nothing happens afterwards.

**Comment by thomas694** [comment]
I'll have a look on it the next days.

Can you give me some more information?
Which Windows 10 version are you using? Is your user account an administrator or normal user?
Please verify that no firewall, antivirus or "internet security suite" is blocking the program.

**Comment by amorakfd** [comment]
Hello Thomas,

Here are the infos requested. PC system is Windows 10 family version 1703, system 15063.0
I don't have any firewall or antivirus or security software which could block the program.

**Comment by thomas694** [comment]
Do you have the same problem with other programs/exe too?
Have you already tried to create a new windows user account and run the exe under that one?

**Comment by amorakfd** [comment]
Hello Thomas,

I worked deeply on the problem these last days and found that there must have been some of windows components not working well or missing. I needed to re-install whole system and some drivers. After that, I am now able to launch the latest TumblThree v1.4. and solve issue with the downloading in the same time.

Blogs are crawled ok, as usual, in queue ;) What a relief to find all its functionalities back again.

Thanks Thomas for your precious advice.
